doc/how-to/contribute.rst

```diff
@@ -0,0 +1,150 @@
+.. _howto-contribute:
+
+How to contribute to MicroCloud
+===============================
+
+The MicroCloud team appreciates contributions to the project, through pull requests, issues on the GitHub repository, or discussions or questions on the forum.
+
+Check the following guidelines before contributing to the project.
+
+Code of Conduct
+---------------
+
+When contributing, you must adhere to the Code of Conduct.
+MicroCloud follows the `Ubuntu Code of Conduct`_.
+
+License and copyright
+---------------------
+
+By default, any contribution to this project is made under the AGPL-3.0 license.
+See the `license`_ in the MicroCloud GitHub repository for detailed information.
+
+The author of a change remains the copyright holder of their code (no copyright assignment).
```

Comment:
This is based on LXD. Is it correct for MicroCloud?

Reply:
I believe so, but not sure because of the CLA. @escabo does this seem OK?

Reply:
Thanks! I'll add info about the CLA here - I missed that it was required.